src/api/hapi/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+
+FILE(GLOB hapi_headers
+    "*.hpp"
+    "*.h")
+
+FILE(GLOB hapi_sources
+    "${CMAKE_CURRENT_SOURCE_DIR}/*.cpp")
+
+ADD_LIBRARY(af SHARED
+            ${hapi_headers}
+            ${hapi_sources})
+
+IF(${BUILD_CPU})
+    ADD_DEPENDENCIES(af afcpu)
+ENDIF()
+
```

Comment:
Technically hapi does not need to wait for any backend to be built. There is no reason for this.